src/metpy/calc/cross_sections.py

```diff
@@ -292,12 +292,14 @@ def absolute_momentum(u, v, index='index'):
 
     """
     # Get the normal component of the wind
-    norm_wind = normal_component(u, v, index=index).metpy.convert_units('m/s')
+    u = u.metpy.convert_units('m/s')
+    v = v.metpy.convert_units('m/s')
+    norm_wind = normal_component(u, v, index=index)
 
     # Get other pieces of calculation (all as ndarrays matching shape of norm_wind)
     latitude = latitude_from_cross_section(norm_wind)
     _, latitude = xr.broadcast(norm_wind, latitude)
-    f = coriolis_parameter(np.deg2rad(latitude.values))
+    f = coriolis_parameter(latitude)
     x, y = distances_from_cross_section(norm_wind)
     x = x.metpy.convert_units('meters')
     y = y.metpy.convert_units('meters')
```

Comment:
Could maybe just entirely eliminate these lines? Not quite sure.

Reply:
"It looks like removing them doesn't break any tests"... famous last words.

Reply:
😆 It's an "opportunity" to test better if that's the case. 😉